l1-contracts/src/core/libraries/validator-selection/ValidatorSelectionLib.sol

```diff
@@ -268,8 +271,8 @@ library ValidatorSelectionLib {
    *
    * @dev     The `_epoch` will never be 0 nor in the future
    *
-   * @dev     The return value will be equal to keccak256(n, block.prevrandao) for n being the last epoch
-   *          setup.
+   * @dev     The return value will be equal to keccak256(n, block.prevrandao) for n being the
+   *          penultimate epoch setup.
```

Comment:
Have never seen penultimate before. 
<img width="654" alt="image" src="https://github.com/user-attachments/assets/35266eb9-ec1d-4878-a83c-bf433b270c81" />

Reply:
Yet you had seen Gerousia